### ISSUE TITLE
White screen when opening Leapp

### ISSUE THREAD

**Issue by AndreVelde** [Noovolari/leapp#524]
**Describe the bug**
When I open leapp I get a completely white screen and cant do anything (it worked before but a couple months back it stopped working and never worked afterwards)

**Leapp Version**
0.24.4 but also tested on older versions

**To Reproduce**
1. Open leapp

**Expected behavior**
I get a completely white screen and cant do anything

**Screenshots**
![leapp issue](https://github.com/Noovolari/leapp/assets/77978453/897de8cb-2e2a-4239-8243-05d76cd5db04)

**Desktop (please complete the following information):**
 - OS: Windows
 - OS Version 10
 - Leapp Version 0.24.4

**Additional context**
Have already tried to delete the appdata folder and that didnt result in any difference. Also no logs are being created at `C:\Users\<USER>\AppData\Roaming\Leapp\log.electronService.log`

**Comment by ericvilla** [comment]
Hi @AndreVelde, we just tested it again using Windows 10. It's working. I know that providing more contextual information is hard without logs. Therefore, I suggest you to backup your Leapp-lock.json file in the ~/.Leapp directory. Then, restart the desktop app. If the problem does not persist, it means it was related to the workspace configuration

**Comment by AndreVelde** [comment]
I have just removed the folder in Roaming again, the .leapp folder and uninstalled Leapp and installed the newest version of Leapp 0.24.6 but still a white screen. Only thing I could say that I am working from a company laptop with certain company policies on it. Not sure if that can interfere with it. Otherwise not sure what else I can delete or do to get it working again. Otherwise I guess I have to do without Leapp

**Comment by ericvilla** [comment]
Hi @AndreVelde, you can find the .Leapp folder - containing the configuration of your workspace - under your home directory, not the Roaming one. Please, try to close the desktop app and rename the $HOME/.Leapp/Leapp-lock.json as $HOME/.Leapp/Leapp-lock.json.bkp. Run Leapp again; it should create a new $HOME/.Leapp/Leapp-lock.json file

**Comment by AndreVelde** [comment]
Yeah meant that I also deleted the one at $HOME/.Leapp but when I start up Leapp again it is not being created anymore

**Comment by mcooper-gb** [comment]
I have the same issue with v0.26.1 running on Windows 11. 
I have no `C:\Users\<USER>\.Leapp` folder.
I also have no log files in C:\Users\<USER>\AppData\Roaming\Leapp\
I've tried multiple reinstalls.

**Comment by ericvilla** [comment]
@mcooper-gb have you managed to fix the issue? @MattiaCostamagna uses the same setup (v0.26.1 running on Windows 11) without problems

**Comment by mcooper-gb** [comment]
@ericvilla  I just tried another clean install of Leapp on the same machine, running the installer as Administrator using v0.26.1 and I still get the same problem.

**Comment by ORuessel** [comment]
I'm seeing the same **white screen** on startup (window opens, entire client area stays blank).

**Setup:** Ubuntu **22.04** (Jammy), **X11**, corporate **VDI**. Leapp Desktop **0.26.1** (`.deb`, `/opt/Leapp`).

**What I tried:** `--disable-gpu --disable-gpu-sandbox` (and related env vars); also booted kernel **6.8.0-106** and **6.8.0-107** — **no change**, UI still white. So this doesn’t look kernel-only on my side.

**DevTools (renderer):** Remote debugging shows the app loading  
`file:///opt/Leapp/resources/app.asar/dist/leapp-client/index.html`, then failing during Angular bootstrap with:

**Comment by mcooper-gb** [comment]
On another machine, I was able to reproduce this by having a certain PATH env variable - nothing unsual although I cannot remember the details - sorry

**Comment by pethron** [comment]
Do you remember, by any chance, if it was an ENV variable related to NodeJS or something totally unrelated?

**Comment by mcooper-gb** [comment]
That rings a bell, yes.
